docs/instruments/instrument_sql/Meta/aosi_meta.sql

```diff
@@ -0,0 +1,9 @@
+INSERT INTO test_names (Test_name, Full_name, Sub_group) VALUES('aosi', 'AOSI', 1);
```

Comment:
SELECT 'aosi', 'AOSI', ID FROM test_subgroups WHERE Subgroup_name = 'Instruments';

Reply:
good idea ! (same for all below thx for typing them all for me)